packages/block-library/src/embed/test/index.native.js

```diff
@@ -153,6 +177,7 @@ beforeEach( () => {
 	// Mock embed responses
 	mockEmbedResponses( [
 		RICH_TEXT_EMBED_SUCCESS_RESPONSE,
+		RICH_TEXT_EMBED_ERROR_RESPONSE,
```

Comment:
`mockEmbedResponses` uses the `url` property of the response to match the request's URL, so in this case, we shouldn't add `RICH_TEXT_EMBED_ERROR_RESPONSE ` as its value is `null`.

Reply:
Alternatively, you could mock the response to the oEmbed endpoint with a null response within the "Cannot embed should be shown on the placeholder if EmbedPreview data is null" test case:
```
// Return null response for requests to oembed endpoint.
fetchRequest.mockImplementation( ( { path } ) => {
	const isEmbedRequest = path.startsWith( '/oembed/1.0/proxy' );
	return Promise.resolve( isEmbedRequest ? EMBED_NULL_RESPONSE : {} );
} );
```

Reply:
I've addressed the issue in [this commit](https://github.com/WordPress/gutenberg/pull/35533/commits/b6752611f3f767019882e5a88f4f917cd3633091).